### PR TITLE
Publish driftctl to Arch user repository

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #405 
| ❓ Documentation  | no

## Description

Add automatic publish to arch user repository.

Example pipeline can be found there : https://app.circleci.com/pipelines/github/cloudskiff/driftctl/1171/workflows/672dbdd4-d2cb-4ad1-aefe-4af1559288e4/jobs/1845

### CODE REVIEW

**Comment by eliecharra** [comment]
@wbeuil @moadibfr It was quite complicated, attentive review required there 🙏🏻

**Review by wbeuil — CHANGES_REQUESTED** [review]
LGTM as for the script and the ci config, I tested it with a clone repo. Too bad there is not an easier way though ... Just two small comments about the badges

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <img src="https://circleci.com/gh/cloudskiff/driftctl.svg?style=shield"/>
   <img src="https://goreportcard.com/badge/github.com/cloudskiff/driftctl"/>
   <img src="https://img.shields.io/github/license/cloudskiff/driftctl">
-  <img src="https://img.shields.io/github/v/release/cloudskiff/driftctl">
+  <img src="https://img.shields.io/github/v/release/cloudskiff/driftctl?label=latest%20version">
```
[the remark this answers is not in the document]
Comment (eliecharra) [reply README.md:9]: Yeah I think it was unclear with the new block label, will revert this with the usage of individual labels

File: README.md [hunk 2]
```diff
@@ -18,6 +18,10 @@
   <a href="https://discord.gg/NMCBxtD7Nd">
     <img src="https://img.shields.io/discord/783720783469871124?color=%237289da&label=discord&logo=discord"/>
   </a>
+  <br>
+  <a href="https://repology.org/project/driftctl/versions">
+    <img src="https://repology.org/badge/vertical-allrepos/driftctl.svg" alt="Packaging status">
+  </a>
```
[the remark this answers is not in the document]
Comment (eliecharra) [reply README.md:24]: And what about something like this ?

<details>
  <summary>Packages</summary>
    <a href="https://repology.org/project/driftctl/versions">
    Packaging status
  </a>
</details>

[the remark this answers is not in the document]
Comment (wbeuil) [reply README.md:24]: Didn't know we could do that in markdown ! I'm ok with that

**Review by wbeuil — APPROVED** [review]

**Pull request merged**